compareECLFiles.cmake

```diff
@@ -395,7 +398,8 @@ add_test_compareECLFiles(CASENAME wsegsicd
 			  FILENAME TEST_WSEGSICD
 			  SIMULATOR flow
 			  ABS_TOL ${abs_tol}
-			  REL_TOL ${rel_tol})
+			  REL_TOL ${rel_tol}
+              TEST_ARGS --tolerance-pressure-ms-wells=10. --tolerance-wells=1.e-6 --solver-max-time-step-in-days=5.0) 
```

Comment:
This one seems tricky, I suggest we go back to the drawing board on it, and leave it out of the pull request.